modules/obr/dependency-download/build.gradle

```diff
@@ -46,6 +47,14 @@ task getFrameworkDependency {
     }
 }
 
+// Gets the framework dependency and renames it to remove the version number.
```

Comment:
I think this is meant to say extensions here

Reply:
fixed